cidc_api/models/schemas.py

```diff
@@ -0,0 +1,77 @@
+from functools import wraps
+from typing import Optional
+
+from marshmallow import fields
+from marshmallow_sqlalchemy import SQLAlchemyAutoSchema
+
+from ..config.db import db
+from .models import (
+    BaseModel,
+    UploadJobs,
+    Users,
+    DownloadableFiles,
+    Permissions,
+    TrialMetadata,
+)
+
+
+class BaseSchema(SQLAlchemyAutoSchema):
+    class Meta:
+        sqla_session = db.session
+        include_fk = True
+        load_instance = True
+
+
+class _ListMetadata(BaseSchema):
+    total = fields.Int(required=True)
+    # TODO: do we need these fields?
+    # page_num = fields.Int(required=True)
```

Comment:
wouldn't those be useful for showing pagination in the UI?

Reply:
We don't currently use this in the UI implementation, so I'm not convinced it's useful.